Source/WebKit/WebProcess/WebPage/WebInspectorUI.cpp

```diff
@@ -92,12 +92,15 @@ void WebInspectorUI::updateConnection()
     IPC::Connection::Identifier connectionIdentifier, connClient;
     IPC::Connection::createServerAndClientIdentifiers(connectionIdentifier, connClient);
     IPC::Attachment connectionClientPort(connClient);
+#elif PLATFORM(HAIKU)
+	IPC::Connection::Identifier connectionIdentifier;
+	IPC::Attachment connectionClientPort;
 #else
     notImplemented();
     return;
 #endif
 
-#if USE(UNIX_DOMAIN_SOCKETS) || OS(DARWIN) || PLATFORM(WIN)
+#if USE(UNIX_DOMAIN_SOCKETS) || OS(DARWIN) || PLATFORM(WIN) || PLATFORM(HAIKU)
```

Comment:
i have initialized to null in here because we need not worry about UI inspector at the moment

Reply:
Yes, we can't disable it because it's often breaking the build otherwise. But we don't currently support it.